### PR TITLE
Step3

### PR DESCRIPTION
**Pull request by ADL175**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # pyramid-learning-journal
 
+<h1>Step 3</h1>
+
+Implemented SQLalchemy and templating on home and detail/post page. Created environmental variables to deploy to Heroku. 
+
 <h1>Step 2</h1>
 
 Step 2 w/ Carlos partnered on the journal server project. Implemented journal entries and incorporated templating to streamline user experience and refactor unnecessary code.
```
Comment (pasaunders) [inline README.md]: Good to see you keeping the readme up to date, but try to make it something that describes what your code does, not what you did to your code.

Comment (pasaunders) [inline README.md]: You need a Heroku deployment link here.

File: pyramid_learning_journal/pyramid_learning_journal/tests.py [hunk 2]
```diff
@@ -1,78 +1,83 @@
...
+#     html = response.html
+#     import pdb; pdb.set_trace()
+#     assert
 
 
-def test_detail_route_with_bad_id(testapp):
-    """Test the detail route with a bad ID."""
-    response = testapp.get('/journal/400', status=404)
-    assert "Alchemy scaffold" in response.text
+    # config.add_route('detail', '/journal/{id:\d+}')
```
Comment (pasaunders) [inline pyramid_learning_journal/pyramid_learning_journal/tests.py]: Your tests error out. Get the tests working and add a lot more. I want to see tests for expected function, expected failure, and edge cases.

File: pyramid_learning_journal/pyramid_learning_journal/views/default.py [hunk 3]
```diff
@@ -1,39 +1,26 @@
...
-    journal = None
-    for item in JOURNALS:
-        if item['id'] == the_id:
-            journal = item
-            break
-
-    # journal = list(filter(lambda item: item['id'] == the_id, JOURNALS))
-
+    session = request.dbsession
+    journal = session.query(Journal).get(the_id)
```
Comment (pasaunders) [inline pyramid_learning_journal/pyramid_learning_journal/views/default.py]: Missing add and edit views. They're only supposed to return empty dictionaries right now, but they should still be here.

**Review by pasaunders — CHANGES_REQUESTED** [review]
It looks like you've got the core of a decent site here. You just need to get it tested, and expand to fill the spec. I want to see the heroku deployment too.

**Comment by ADL175** [comment]
fixed step3 and merged to master
